client/app/components/pages/GeneHome.vue

```diff
@@ -3911,10 +3926,9 @@ export default {
       let self = this;
       return new Promise(function(resolve, reject) {
         if (theVariant.mosaic_id && theVariant.mosaic_id != "") {
+          cosole.log("looking up mosaic variant by id " + theVariant.mosaic_id)
```

Comment:
This should result in a runtime error due to the misspelling. Also, it is best to leave debugging code like this out of the production code as it fills up the console with unneeded text.